Automattic/wp-calypso
# Domains: Create new edit domain contact info page stub

### Changes proposed in this Pull Request

This PR creates a new `EditContactInfoPage` component which will be the redesigned edit contact info page. It's just a stub for now and was basically a copy-paste of the original `EditContactInfo` component (`/my-sites/domains/domain-management/edit-contact-info/index.jsx`) - this will help us work on the redesign in parallel.

This is part of the domain management redesign project described in pcYYhz-m2-p2.

### Testing instructions

- Build this branch locally or open the live Calypso link
- Go to "Upgrades > Domains"
- Select one of your registered domains
- Select the "Update your contact information" option and then "Edit contact info"
- If you're in the live Calypso link, please append `?flags=domains/contact-info-redesign` to your URL to enable the appropriate feature flag
- Ensure a blank page with white background and a "Page goes here" message is rendered
